### DISCUSSION TITLE
KWGT

### DISCUSSION THREAD

**Discussion by Benzauber** [Acclorite/book-story#135] [Q&A]
Hello,

I'm wondering if there's a way to access the reading progress percentage from Book-Story. I'd like to use this data in a KWGT widget to display how far I am in a book. 
Is there any API or method available to retrieve this information? 

Thank you for your help!"

Benzauber

**Comment by Acclorite** [comment]
No, it is in database and there's not much you can do. You can follow #23, it may solve your question.
